test/validation.test.ts

```diff
@@ -0,0 +1,586 @@
+/* tslint:disable:max-file-line-count */
+
+import { validate } from '../src/validation';
+import {
+  schemaRule,
+  uischemaCategorization,
+  uischemaControl,
+  uischemaGroup,
+  uischemaHorizontalLayout,
+  uischemaVerticalLayout
+} from './exampleUISchemas/validSchemas';
```

Comment:
Use star import : `* as valid from ...`
References to actual schemas then can be prefixed with `valid` which is more explicit.